packages/dai-plugin-scd/src/tokens/Erc20Token.js

```diff
@@ -59,7 +61,8 @@ export default class Erc20Token {
 
   approveUnlimited(spender, options = {}) {
     if (!spender) spender = this._web3.currentAddress();
-    return this._contract.approve(spender, -1, {
+    return this._contract.approve(spender, ethers.BigNumber.from(UINT256_MAX), {
```

Comment:
can no longer approve `-1` as a shortcut for unlimited in ethers 5.

Reply:
i tried to find documentation for this but i can not

Reply:
@rafinskipg [in web3](https://web3js.readthedocs.io/en/v1.2.11/web3-utils.html#id21) it seems a negative number is interpreted as `int256`. Maybe ethers v3 accepted this as well? But you can't send a negative number in EVM [according to this](https://stackoverflow.com/a/52821291).